target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-d7800.dts

```diff
@@ -161,13 +161,45 @@
 
 	pinctrl-0 = <&usb0_pwr_en_pins>;
 	pinctrl-names = "default";
+	
+	bridge@0,0 {
+		reg = <0x00000000 0 0 0 0>;
+		#address-cells = <3>;
+		#size-cells = <2>;
+		ranges;
+
+		wifi@1,0 {
+			compatible = "pci168c,0040";
+			reg = <0x00010000 0 0 0 0>;
+
+			nvmem-cells = <&macaddr_art_6>;
+			nvmem-cell-names = "mac-address";
+			mac-address-increment = <(1)>;
+		};
+	};
 };
 
 &usb3_1 {
```

Comment:
@Ansuel Are these two attached to usb nodes by intention?!

Reply:
not at all... i assume a rebase problem

Reply:
I've already pulled it to my local tree, so no need to rebase for you. Just wanted to know. I'll fix it here.